### PR TITLE
Chris/nearby pets

### PR DESCRIPTION
**Pull request by hehachris**
- [x] Endpoint to find nearby pets
- [x]  Google map to show nearby pets with markers

@anthonylau pls help review

### CODE REVIEW

File: app/models/Pet.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ const Pet = sequelize.define('pet', {
         type: Sequelize.BOOLEAN,
         allowNull: false,
         defaultValue: false
+    },
+    latitude: {
+        type: Sequelize.DECIMAL(10, 8),
+        allowNull: true
+    },
+    longitude: {
+        type: Sequelize.DECIMAL(11, 8),
```
Comment (anthonylau) [inline app/models/Pet.js:43]: just curious why lat is 10 lon is 11

File: app/repos/sequelize/pet.js [hunk 2]
```diff
@@ -36,5 +38,19 @@ module.exports = {
         return models.Pet.findAll({
             where: criteria
         });
+    },
+    findNearby(lat, lng, km = 2) {
+        return this.findAll()
```
Comment (anthonylau) [inline app/repos/sequelize/pet.js]: find all will hit scale problem?

File: app/models/Pet.js (hunk 1 above)
Comment (hehachris) [reply app/models/Pet.js:43]: lat ranges from -90 to 90
lng ranges from -180 to 180

File: app/repos/sequelize/pet.js (hunk 2 above)
Comment (hehachris) [reply app/repos/sequelize/pet.js]: Good point. Updated to query pets only within boundary.

**Pull request merged**